processing/src/main/java/io/druid/query/aggregation/cardinality/CardinalityAggregatorFactory.java

```diff
@@ -213,17 +209,16 @@ public AggregatorFactory getMergingFactory(AggregatorFactory other) throws Aggre
   @Override
   public List<AggregatorFactory> getRequiredColumns()
   {
-    return Lists.transform(
-        fields,
-        new Function<DimensionSpec, AggregatorFactory>()
-        {
-          @Override
-          public AggregatorFactory apply(DimensionSpec input)
-          {
-            return new CardinalityAggregatorFactory(input.getOutputName(), Collections.singletonList(input), byRow);
-          }
-        }
-    );
+    return fields.stream()
+                 .map(
+                     input -> new CardinalityAggregatorFactory(
+                         input.getOutputName(),
+                         null,
+                         Collections.singletonList(input),
+                         byRow,
+                         round
+                     )
+                 ).collect(Collectors.toList());
```

Comment:
Line should be breaked after `)`

Reply:
Says who? :P

I'm using the IntelliJ codestyle config (druid_intellij_formatting.xml) from the Druid repo and it formatted this line the way you see it. I usually consider that formatter + the automated checkstyle as the final word on style.

I changed this (don't want to argue) and am fine with going with a different style in general, but can you please add the rule to something automated, either intellij config or checkstyle?

Reply:
I didn't find options neither in IntelliJ nor in Checkstyle to enforce this rule.

That being said IntelliJ formatting is pretty bad and often produces really ugly constructions